SerialPrograms/CMakeLists.txt

```diff
@@ -1259,10 +1261,10 @@ file(GLOB MAIN_SOURCES
     Source/PokemonSV/Options/PokemonSV_AuctionItemTable.h
     Source/PokemonSV/Options/PokemonSV_PlayerList.cpp
     Source/PokemonSV/Options/PokemonSV_PlayerList.h
-    Source/PokemonSV/Options/PokemonSV_TeraAIOption.cpp

```

Comment:
Can also revert this since someone (usually Gin) probably ran a reordering script to put them in alphabetical order.